data-visualization/accessibility.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: page
+title:  Accessibility
+category: Data visualization
+published: true
+---
+
+- [Alt tags](#alt-tags)
+- [Color contrast and color blindness](#color-contrast-color-blindness)
+ {: class="toc"}
+
+
+Allowing all users to be able to comprehend charts and graphics is a key part
+of being a government agency that serves the entire American public. This is an
+interpretation of what it means for a chart to be 508 compliant and accessible,
+starting with descriptive titles (see the [Data visualization text
+  section](text.html)) and not only relying on color to connect data to its
+  meaning (see the [Data visualization color section](color.html)).
+{: class="lead-in"}
+
+<h5>Key questions</h5>
```

Comment:
In general for the headings that don't require anything special, you can use markdown headings and save yourself from entering tags and ids:

`##### Key questions`

This includes headings that need an id but nothing else, like the h3 on L:25. Github will add the ids automatically, so these should be:

`### Alt tags`